iOS/SceneCoordinator.swift

```diff
@@ -1183,6 +1184,21 @@ class SceneCoordinator: NSObject, UndoableCommandRunner, UnreadCountProvider {
 		}
 		UIApplication.shared.open(url, options: [:])
 	}
+
+	func showSafariForCurrentFeed() {
+		if let ip = currentFeedIndexPath, let url = homePageURLForFeed(ip) {
+			let vc = SFSafariViewController(url: url)
+			rootSplitViewController.viewControllers.last?.present(vc, animated: true)
+		}
+	}
+
+	func showSafariForCurrentArticle() {
+		guard let preferredLink = currentArticle?.preferredLink, let url = URL(string: preferredLink) else {
+			return
+		}
+		let vc = SFSafariViewController(url: url)
+		rootSplitViewController.viewControllers.last?.present(vc, animated: true)
```

Comment:
Not sure this is the right way of doing it.

Reply:
I think I would add a function in ArticleViewController (that would then call WebViewController) that can be called to open the SFViewController.

Reply:
Instead of `openInSafari` let's use `openInAppBrowser` instead.  I think that makes it clearer that `openInBrowser` opens externally and `openInAppBrowser` opens the internal browser.

Reply:
Sound good! let me change it